torch/jit/__init__.py

```diff
@@ -789,6 +789,103 @@ def wrap_check_inputs(check_inputs):
 
     return [{'forward' : c} for c in check_inputs]
 
+
+def fork(func,
+          *args,
+          **kwargs):
+    """
+    Creates an asynchronous task executing `func` and return a reference
+    to the execution of this function. `fork` will return immediately,
```

Comment:
In my doc, I referred to the return value as "a reference to the value of the result of this execution", just to emphasize that it's a value contained in the Future